lib/cb/utils/api.rb

```diff
@@ -46,9 +46,10 @@ def cb_delete(path, options={}, &block)
 
       def execute_http_request(http_method, uri, path, options={}, &block)
         self.class.base_uri(uri || Cb.configuration.base_uri)
-        cb_event(:"cb_#{http_method.to_s}_before", path, options, nil, &block)
+        start_time = Time.now.to_f
+        cb_event(:"cb_#{http_method.to_s}_before", path, options, nil, start_time - start_time, &block)
         response = self.class.method(http_method).call(path, options)
-        cb_event(:"cb_#{http_method.to_s}_after", path, options, response, &block)
+        cb_event(:"cb_#{http_method.to_s}_after", path, options, response, Time.now.to_f - start_time, &block)
```

Comment:
Agreed, I didn't even notice that that was there before :smile: Boy scouted some other portions of this though. Lemme get rid of that `.to_s` real quick.